lib/galaxy/jobs/runners/util/job_script/__init__.py

```diff
@@ -43,12 +43,14 @@
 
 # Copy working, outputs, and configs before tool execution so that these can be restored on job resubmission
 # xref https://github.com/galaxyproject/galaxy/issues/3289
-PREPARE_DIRS = """mkdir -p working outputs configs
-if [ -d _working ]; then
-    rm -rf working/ outputs/ configs/; cp -R _working working; cp -R _outputs outputs; cp -R _configs configs
-else
-    cp -R working _working; cp -R outputs _outputs; cp -R configs _configs
-fi
+PREPARE_DIRS_TEMPLATE = """for dir in working outputs configs; do
+    mkdir -p "{working_directory}/_$dir"
```

Comment:
```suggestion
    mkdir -p "{working_directory}/$dir"
```

otherwise the next if condition would always be true.

Reply:
Thanks, I didn't actually run this, just figured the tests (or an :eagle::eyes: reviewer) would catch anything I messed up :sweat_smile: